src/documentation/search/search.ts

```diff
@@ -200,12 +205,20 @@ const apiSearchableContent = (
 export const buildSearchIndex = (
   searchableContent: SearchableContent[],
   tab: "reference" | "api",
+  language: string,
   ...plugins: lunr.Builder.Plugin[]
 ): SearchIndex => {
   const index = lunr(function () {
     this.ref("id");
     this.field("title", { boost: 10 });
     this.field("content");
+    // There is always some degree of English content.
+    const multiLanguages = ["en"];
```

Comment:
I've seen some GitHub issues where using multiple languages causes issues with tokenisation of the main target language. It might be more sensible to throwaway English here unless the main language is not supported. https://github.com/MihaiValentin/lunr-languages/issues

Reply:
Can you try to get to the bottom of this? Is it working for Japanese? I read https://github.com/MihaiValentin/lunr-languages/issues/45 -  are there others that are relevant?

Reply:
I also only found the issue for Japanese. 

After running tests, it looks like we find "日本語" in the following  "この文章は日本語で書かれています" in both the single ja pipeline and the multi-language en and ja pipeline. So the multilang approach is not a problem.

The issue we retain is that searching for "書かれ" returns nothing, but "書か れ" is a match. Indexing splits "書かれ" into "書か" and "れ". I'm not sure how we deal with this.

Reply:
OK, we'll merge this as a big step forward and I'll try to look further at this.